docs/midstream-getting-serveless-operator-bugfixes.md

```diff
@@ -0,0 +1,132 @@
+# Getting bugfixes for midstream Serverless Operator
+
+## 1. Introduction
+
+This document explains how to update the midstream Serverless operator image so that you can get bugfixes.
+
+Serverless Operator midstream, unlike the fully released product, does not create patch releases.
+When we need to fix a bug, we overwrite the existing image with a new one.
+
+For example, when we identify a bug in 1.24 version, we change the code in 1.24 branch and the CI/CD overwrites the `1.24.0` image.
+
+## 2. Prerequisites
+
+- Usage of the midstream Serverless Operator and not the fully released product
+- Access to OCP cluster with installed Serverless Operator
+- Permission to delete pods and replicasets in `openshift-serverless` namespace.
+- If using Knative Eventing and Knative Kafka components, permission to delete pods and replicasets in `knative-eventing` namespace.
+- If using Knative Serving, permission to delete pods and replicasets in `knative-serving`, `knative-serving-ingress` namespaces.
+
+## 3. Execute/Resolution
+
+1. Get the Serverless Operator images used in the OpenShift cluster right now, for later comparison:
+  ```shell
+  > kubectl get pods -n openshift-serverless -o jsonpath='{range .items[*]}{"\n"}{.metadata.name}{": "}{range .status.containerStatuses[*]}{.imageID}{end}{end}' | sort 
+  knative-openshift-5dbb4bd4b7-q6wdp: registry.ci.openshift.org/knative/openshift-serverless-v1.24.0@sha256:26c3127e3dab1e102b94c305916d405b57d397c56348e0d73c7e1c2cddf3310b
+  knative-openshift-ingress-78766bdc5c-zfzfl: registry.ci.openshift.org/knative/openshift-serverless-v1.24.0@sha256:7a417307328f76462560c5d4f814566135e59fc1b20758be720d090047ec682e
+  knative-operator-65487bf7fc-vbgkk: registry.ci.openshift.org/knative/openshift-serverless-v1.24.0@sha256:589735abf033c61247e594f3fba9e7d87ac1093d1ebc0abe73f825944fe5e465
+  ```
```

Comment:
would `kubectl rollout restart deployment ...` work instead of deleting pods?

Reply:
deleting pods might be risky vs rollout because it causes downtime

Reply:
Trying now

Reply:
That seems to work. Updated.